src/super_gradients/recipes/variable_setup.yaml

```diff
@@ -0,0 +1,42 @@
+# Varaible setup for shortcuts and setting the hydra output directory.
+# Any SG Recipe should set this yaml file as a default, after _self_, i.e at the top of your recipe file:
+#
+#  defaults:
+#    - training_hyperparams: my_train_params
+#    - dataset_params: my_dataset_params
+#    - arch_params: my_arch_params
+#    - checkpoint_params: my_checkpoint_params
+#    - _self_
+#    - variable_setup
+#
+#
+#  Interpolates the shortcuts defined below, with their aliases (see comments near each parameter).
+#  When any of the above are not set, they will be populated with the original values (for example
+#      config.lr will be set with config.training_hyperparams.initial_lr) for clarity in logs.
+#
+# Note that interpolation is done by triggering RecipeShortcutsCallbackm which is a Hydra Callback (see http://hydra.cc/docs/experimental/callbacks/)
+#  so interpolation of these in other yaml configuration files won't be present.
+
+lr: # config.training_hyperparams.initial_lr
```

Comment:
I would add an inline example, I think it makes the use more straightforward:

`python train_from_recipe --config-name=recipe lr=0.003`

`python train_from_recipe --config-name=recipe config.training_hyperparams.initial_lr=0.003`

Reply:
NP, added in c2d8e7ca47a3c72b5a5830122728c708f3bd8d30